docs/api-docs/generate_typedoc.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
 
-BASE_DIR="../langchain"
+BASE_DIR="../../langchain"
 # Create a directory to temp hold the files in src
 # `docs_dist` is gitignored by default.
-mkdir -p "$BASE_DIR/docs_dist/src"
+mkdir -p "$BASE_DIR/docs/dist"
```

Comment:
How about `/docs/api-docs/dist` to keep everything localized?

Reply:
Or actually sorry but how about `docs/api_refs/` (we use snake case for directories elsewhere and `docs/api_docs` feels redundant)